src/react-imgix-bg.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import Measure, { withContentRect } from "react-measure";
+import constructUrl from "./constructUrl";
+import targetWidths from "./targetWidths";
+import findClosest from "findClosest";
```

Comment:
This should be a relative import
```suggestion
 import findClosest from "./findClosest";
```

I believe this will search for findClosest in `node_modules/`
![image](https://user-images.githubusercontent.com/6812331/50295114-58c3eb00-0445-11e9-91d5-920f29b0aa43.png)